source/perconaft.rst

```diff
@@ -43,41 +75,7 @@ By default, Percona Server for MongoDB runs with MMAPv1. To enable PerconaFT:
 
 .. warning:: Transparent huge pages is a feature in newer Linux kernel versions that causes problems for the memory usage tracking calculations in PerconaFT and can lead to memory overcommit. If you have this feature enabled, PerconaFT will not start, and you should turn it off. If you want to run with transparent hugepages on, you can set an environment variable ``TOKU_HUGE_PAGES_OK=1``, but only do this for testing, and only with a small cache size.
 
-Storage Engine Comparison
-=========================
-
-|Percona Server for MongoDB| includes several storage engines, which have a different set of features and advantages over one another. To help you decide which is better for you, the table below lists features available in various storage engines:
-
-.. list-table::
-   :header-rows: 1
-   :stub-columns: 1
-
-   * -
-     - MMAPv1
-     - WiredTiger
-     - MongoRocks [#n-1]_
-     - PerconaFT
-   * - Available in MongoDB
-     - YES
-     - YES
-     - NO
-     - NO
-   * - Document-level locking
-     - NO
-     - YES
-     - YES
-     - YES
-   * - High compression
-     - NO
-     - YES
-     - YES
-     - YES
-   * - Fast insertions
-     - NO
-     - NO
-     - YES
-     - YES
-
-.. rubric:: Footnotes
+Using Other Storage Engines
+===========================
 
-.. [#n-1] This feature is currently considered *Experimental*
+|Percona Server for MongoDB| includes several storage engines, which have a different set of features and advantages over one another. You can switch between storage engines using the ``--storageEngine`` option. Also specify the ``--dbpath`` option to a separate data directory for every storage engine you use.
```

Comment:
DOC-233 bug refers to post-install steps, due to changes in package installation script.

With new packaging once you install packages, server will automatically start after installation with the MMAPv1 as the default storage engine and if you try to change the engine simply with --storage-engine option it would fail.

Note should be provided here and more details in the deb installation guide on how to change the default storage engine after packages have been installed. (stop service, rm -rf mongod-datadir, change the storage engine in the mongo.conf, start service)

Docs are missing information on ability to change the storage engine via mongod.conf configuration file.